pkg/pillar/pubsub/pubsubintf.go

```diff
@@ -5,7 +5,7 @@
 
 package pubsub
 
-// Publication - Interface to be implemented by a Publication
+// Publication - Interface to be implemented by a NewPublication
```

Comment:
Same as before; revert comment. A `Publication` implementation implements the interface; `NewPublication` is just the func call.